Networking/Networking/Model/Order.swift

```diff
@@ -189,6 +194,7 @@ private extension Order {
         case shippingTax        = "shipping_tax"
         case total              = "total"
         case totalTax           = "total_tax"
+        case paymentMethodID    = "payment_method"
```

Comment:
At first, I was confused about why we're diverging from the REST API name but after looking [at the docs](https://woocommerce.github.io/woocommerce-rest-api-docs/#order-properties), I believe using `ID` makes more sense!